src/js/components/Section.js

```diff
@@ -2,12 +2,16 @@
 
 var React = require('react');
 var Box = require('./Box');
+var SkipLinkAnchor = require('./SkipLinkAnchor');
+var merge = require('lodash/object/merge');
 
 var CLASS_ROOT = "section";
 
 var Section = React.createClass({
 
-  propTypes: Box.propTypes,
+  propTypes: merge({
+    primary: React.PropTypes.bool
+  }, Box.propTypes),
```

Comment:
I feel we need to flip this order for the arguments. Section propTypes should take over Box propTypes.